### ISSUE TITLE
Feature for printing assets with json output option

### ISSUE THREAD

**Issue by microwaves** [kubernetes/kops#10317]
As discussed on Slack, I would like to request a feature for listing and printing assets required by Kops, similar to what's done executing `kops update cluster xyz.cluster.com --phase assets -v4`.

I believe this feature could be helpful for parsing and making sure those assets are available before creating/updating the cluster. As in private network setups, non-docker hub setups (Docker Hub rate limit), clusters in China, and so on.

Ideally, a json output option would be very neat! :-)

Example:
```
kops toolbox assets --output json
{
  "k8s.gcr.io/cluster-proportional-autoscaler-amd64": "1.4.0",
  "coredns/coredns": "1.6.2",
  "kope/dns-controller": "1.16.4",
  "kope/kops-controller": "1.16.4",
  ...
}
```

Thanks!

**Comment by olemarkus** [comment]
This is a great suggestion. I could imagine using this to prebake images.

I would certainly review a PR on this.

**Comment by johngmyers** [comment]
Fixed by #11617
/close
